doc/manual/src/language/advanced-attributes.md

```diff
@@ -208,12 +208,26 @@ Derivations can declare some infrequently used optional attributes.
     about converting to and from base-32 notation.)
 
   - [`__contentAddressed`]{#adv-attr-__contentAddressed}
-    If this **experimental** attribute is set to true, then the derivation
+    > **Warning**
+    > This attribute is part of an [experimental feature](@docroot@/glossary.md#gloss-experimental-features).
```

Comment:
The link should point to the page, not the glossary. The glossary is an entry point, not the source of truth.

Reply:
```suggestion
    > This attribute is part of an [experimental feature](@docroot@/contributing/experimental-features.md).
```